packages/tables/src/Columns/Concerns/HasAlignment.php

```diff
@@ -15,19 +15,34 @@ public function alignment(string | Closure | null $alignment): static
         return $this;
     }
 
-    public function alignLeft(bool | Closure $condition = true): static
+    public function alignStart(bool | Closure $condition = true): static
     {
-        return $this->alignment(static fn (): ?string => $condition ? 'left' : null);
+        return $this->alignment(static fn (): ?string => $condition ? 'start' : null);
     }
 
     public function alignCenter(bool | Closure $condition = true): static
     {
         return $this->alignment(static fn (): ?string => $condition ? 'center' : null);
     }
 
+    public function alignEnd(bool | Closure $condition = true): static
+    {
+        return $this->alignment(static fn (): ?string => $condition ? 'end' : null);
+    }
+
+    public function alignJustify(bool | Closure $condition = true): static
+    {
+        return $this->alignment(static fn (): ?string => $condition ? 'justify' : null);
+    }
+
+    public function alignLeft(bool | Closure $condition = true): static
```

Comment:
@danharrin, shouldn't we mark the old methods as deprecated?

Reply:
Nah I think they're usable by people who dont ever support RTL

Reply:
But still they're confusing for people only supporting RTL.

Reply:
I think in some case we need this option, for ex mobile, tel or fax is starting from left for both RTL and LTR

Reply:
Oh thanks for the insights, @AAbosham! Definitely have to keep those then.

Reply:
We'll need to fix the implementation then because I added them as aliases

Reply:
#4853 https://github.com/filamentphp/filament/pull/4853